src/libs/runtime-utils.js

```diff
@@ -116,3 +116,7 @@ export const appIsSettingUp = app => {
 export const collapsedRuntimeStatus = runtime => {
   return runtime && (runtime.patchInProgress ? 'LeoReconfiguring' : runtime.status) // NOTE: preserves null vs undefined
 }
+
+export const getAppStatus = app => {
```

Comment:
This doesn't appear to be used anywhere, should it be deleted?

Reply:
Removed this from the code